src/mainmenu.cpp

```diff
@@ -12,6 +12,7 @@ MainMenu::MainMenu(QWidget *parent) :
 
     ADD_ACTION(KiwixServeAction);
     ADD_ACTION(RandomArticleAction);
+    ADD_ACTION(OpenHomePageAction);
```

Comment:
I would prefer to not add the button to the menu.

- It is not in the design
- The button is not is not align.

![Capture d’écran du 2019-06-12 10-21-08](https://user-images.githubusercontent.com/86161/59335294-96227a80-8cfc-11e9-8467-a21251ccfe36.png)

It is far simpler to simply add the action to the main window.